src/snoowrap.js

```diff
@@ -233,6 +233,32 @@ const snoowrap = class snoowrap {
       return requester;
     });
   }
+  static fromInstalledClientAuth ({
```

Comment:
Can you add JSDoc documentation for this method? The block comments above methods are automatically processed and used to generate the documentation at https://not-an-aardvark.github.io/snoowrap. (You can look at the comments above other methods for examples.)